ws.py

```diff
@@ -453,7 +453,7 @@ def indicators(region):
 def get_ds(ISO):
     if ISO not in available_region_files:
         raise ValueError(f"ISO {ISO} not found")
-    fn = f"/data/DataUpdate_02_2024/xr_alloc_{ISO}.nc"
+    fn = CABE_DATA_DIR / "xr_alloc_{ISO}.nc"
```

Comment:
Should be f-string, otherwise country page doesn't work
```suggestion
    fn = CABE_DATA_DIR / f"xr_alloc_{ISO}.nc"
```